test/suites/consolidation/suite_test.go

```diff
@@ -0,0 +1,165 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package consolidation
+
+import (
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	corev1beta1 "sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	"sigs.k8s.io/karpenter/pkg/test"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/karpenter-provider-azure/test/pkg/debug"
+	environmentazure "github.com/Azure/karpenter-provider-azure/test/pkg/environment/azure"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var env *environmentazure.Environment
+
+func TestConsolidation(t *testing.T) {
+	RegisterFailHandler(Fail)
+	BeforeSuite(func() {
+		env = environmentazure.NewEnvironment(t)
+	})
+	RunSpecs(t, "Consolidation")
+}
+
+var _ = BeforeEach(func() { env.BeforeEach() })
+var _ = AfterEach(func() { env.Cleanup() })
+var _ = AfterEach(func() { env.AfterEach() })
+
+var _ = Describe("Consolidation", func() {
+	It("should consolidate nodes (delete)", Label(debug.NoWatch), Label(debug.NoEvents), func() {
+		provider := env.DefaultAKSNodeClass()
+		nodePool := env.DefaultNodePool(provider)
+		nodePool.Spec.Disruption.ConsolidationPolicy = corev1beta1.ConsolidationPolicyWhenUnderutilized
+		nodePool.Spec.Disruption.ConsolidateAfter = &corev1beta1.NillableDuration{}
+
+		var numPods int32 = 75
```

Comment:
We've been typically trying to keep numbers lowers for tests atm. I think 75 pods requiring 1 cpu is far beyond what we currently want for testing. I'd suggest something < 20.

Reply:
My req would actually be to use something closer to 10

Reply:
Resolving since we removed consolidation